app/package.json

```diff
@@ -16,7 +16,10 @@
     "react-i18next": "^11.7.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.3",
-    "typescript": "^3.8.0"
+    "typescript": "^3.8.0",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1"
```

Comment:
I think there's no need for those dependencies (body-parser, cors and express) in this `package.json` as it is the frontend package.json.

Try to remove then and check whether the scripts keep working as supposed

Reply:
sure @tiagostutz   earlier- without these dependencies - the script was not working ,let me try that out again

Reply:
@tiagostutz any update ??

Reply:
hey @tiagostutz  i was tar-getting to close this in October it-self ,so it will be great if you can help me in closing this PR

Reply:
Sorry about late merge, @shalika10... =/

I just merged the PR.! Thanks a lot!